lib/deck.rb

```diff
@@ -0,0 +1,24 @@
+class Deck
+
+  attr_reader :cards
+  def initialize(cards)
+     @cards = cards
+
+  end
+
+  def count
+    cards.count
+  end
+
+  def cards_in_category(category)
+    test = []
+    @cards.each do |card|
+      # test << category == card.category
+      if category == card.category
+        test << card
+      end
+    end
+    return test
```

Comment:
I see you used an accumulator here - great - try to be more descriptive with your variable names. What does `test` tell me about the data contained within? (Answer: not much)